### DISCUSSION TITLE
Downtime post-mortem

### DISCUSSION THREAD

**Discussion by flickz** [codesandbox/codesandbox-client#4208] [General] [answered]
Yesterday, [Codesandbox](https://codesandbox.io/) was down for some minutes:

<img width="606" alt="Screenshot 2020-05-26 at 11 18 57 AM" src="https://user-images.githubusercontent.com/15707013/82889492-b9acf800-9f42-11ea-8394-4aee0f4f0920.png">

Is there any postmortem report on what happened?

**Comment by garethx** [comment, marked as answer]
Here you go: https://twitter.com/codesandbox/status/1265032904066899969?s=20

**Reply by flickz** [reply]
Thanks
